webextensions/manifest/content_security_policy.json

```diff
@@ -39,6 +39,7 @@
               "edge": "mirror",
               "firefox": {
                 "version_added": "72",
+                "notes": "From Firefox 105, the directive <code>object-src<code> is not supported.",
```

Comment:
```suggestion
                "notes": "Up until Firefox 104, the <code>object-src</code> directive was required with a secure source. Since Firefox 105, the <code>object-src</code> directive is optional.",
```

Reply:
@Rob--W could you check again, I think I got the version wrong. Also "since" may be ambiguous as it also means "as" etc. so changed.

Reply:
@rebloor You edited @Rob--W's suggestion to change "Since Firefox 105" to "106", but you didn't tweak "Until Firefox 104".  What happens in Firefox 105?  Does the other part need to be modified as well?

By the way, @rebloor, please avoid modifying others' comments in such ways unless agreed upon or otherwise needed to remove language that violates code of conduct.  If there's a discrepancy in the version numbers, it's better to add your own comment so that it can be further discussed.  :)

Reply:
Ah, I see that this suggestion wasn't merged directly and instead another commit was made to make the requested changes with an answer to my question, never mind!